python/py-poetry-core/Portfile

```diff
@@ -5,17 +5,17 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-poetry-core
-version             1.0.8
+version             1.1.0
```

Comment:
This version no longer works with python 3.6. It looks like nothing is using the 36 subport, so it can just be dropped.